helm/hpcc/templates/localroxie.yaml

```diff
@@ -44,7 +44,7 @@ data:
 {{- if not $roxie.disabled  -}}
 {{- $env := concat ($.Values.global.env | default list) (.env | default list) -}}
 {{- $secretsCategories := list "system" "ecl-user" "ecl" "storage" }}
-{{- $commonCtx := dict "root" $ "me" $roxie "secretsCategories" $secretsCategories "env" $env }}
+{{- $commonCtx := dict "root" $ "me" $roxie "includeCategories" (list "lz" "data" "spill") "secretsCategories" $secretsCategories "env" $env }}
```

Comment:
Shouldn't this include "dll", and the call to addDllVolumeMount be removed (and all other places that use addDllVolumeMount/addDllVolume)?

Reply:
I thought about it, but deliberately stopped at going that far with changes, and only focussed on "data" planes (lz, data, spill), leaving helpers in place/use.
But I could go further.